1-org/variables.tf

```diff
@@ -67,6 +67,23 @@ variable "data_access_table_expiration_ms" {
   default     = 2592000000
 }
 
+variable "scc_notification_name" {
+  description = "Name of SCC Notification"
+  type        = string
+}
+
+variable "skip_gcloud_download" {
+  description = "Whether to skip downloading gcloud (assumes gcloud is already available outside the module)"
+  type        = bool
+  default     = true
+}
+
+variable "scc_notification_filter" {
+  description = "Filter used to SCC Notification, you can see more details how to create filters in https://cloud.google.com/security-command-center/docs/how-to-api-filter-notifications#create-filter"
+  type        = string
+  default     = "state=\\\"ACTIVE\\\""
```

Comment:
Are all of these backslashes required? The documentation seems to only require one